pkg/utils/status.go

```diff
@@ -242,12 +244,25 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 	log.Infof("daemonset expected [%d]", expected)
 	log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
 
-	if (controllerReplicas == controllerStatus.Available) && (fmt.Sprintf("%d", expected) == nodeStatus.Available) {
-		running = true
-		newStatus.State = constants.Succeeded
+	if instance.GetName() == "application-mobility" {
+		modrunning, err := statusForAppMob(ctx, instance, r, newStatus)
+		if err != nil {
+			log.Infof("calculate Daemonseterror msg [%s]", err.Error())
```

Comment:
Can we change this error message to
"statusForAppMob err msg.."

Reply:
sure